examples/puppet/modules/install_otel_collector/manifests/init.pp

```diff
@@ -62,21 +64,36 @@
 
     # Run install script with tags defined inside PowerShell command
     exec { 'Install Otel Collector':
-      command => "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe -ExecutionPolicy RemoteSigned -Command \"\
+      command   => "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe -ExecutionPolicy Bypass -Command \"\
         Set-ExecutionPolicy RemoteSigned -Scope Process -Force; \
```

Comment:
Why the change from `RemoteSigned` to `Bypass`? That doesn't seem related to fixing a Puppet error.